### PR TITLE
MGDOBR-537: Code coverage using jacoco

### PR DESCRIPTION
**Pull request by Aakanksha0311**
[MGDOBR-537](https://issues.redhat.com/browse/MGDOBR-537)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: kustomize/overlays/local/istio/jwt-request-authentication.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ metadata:
 spec:
   jwtRules:
     - issuer: "http://<REPLACE_WITH_MINIKUBE_IP>:30007/auth/realms/event-bridge-fm"
-      jwksUri: "http://keycloak.keycloak:8180/auth/realms/event-bridge-fm/protocol/openid-connect/certs"
+      jwksUri: "http://keycloak.keycloak:8180/auth/realms/event-bridge-fm/protocol/openid-connect/certs"
```
Comment (jomarko) [inline kustomize/overlays/local/istio/jwt-request-authentication.yaml:9]: is this change needed to be part of this PR?

File: build-parent/pom.xml [hunk 2]
```diff
@@ -328,7 +328,7 @@
           <configuration>
             <!-- ${argLine} placeholder needed due to surefire & jacoco issue -->
             <!-- https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven -->
-            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 ${argLine}</argLine>
+            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 @{argLine}</argLine>
```
Comment (jomarko) [inline build-parent/pom.xml:331]: I think we should update also comment? form ${argLine} to @{argLine}?

**Review by jomarko — COMMENTED** [review]
Thank you for PR @Aakanksha0311 . Can be effetc of your changes seen somewhere? Or it is just an 'attempt' to fix?

File: kustomize/overlays/local/istio/jwt-request-authentication.yaml (hunk 1 above)
Comment (Aakanksha0311) [reply kustomize/overlays/local/istio/jwt-request-authentication.yaml:9]: No that is not a part. will fix it

**Comment by Aakanksha0311** [comment]
> Thank you for PR @Aakanksha0311 . Can be effetc of your changes seen somewhere? Or it is just an 'attempt' to fix?

Yes, I tried to check for the coverage report using a draft PR - [(https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1149)]. Here you can see the coverage percentage.

File: build-parent/pom.xml (hunk 2 above)
Comment (Aakanksha0311) [reply build-parent/pom.xml:331]: Sure will do it.

**Review by sutaakar — APPROVED** [review]

**Review by Srihari1192 — APPROVED** [review]

**Review by jomarko — APPROVED** [review]
thank you @Aakanksha0311
